### ISSUE TITLE
PVR Home Sreen Widget Logo/Artwork does not show

### ISSUE THREAD

**Issue by gwx12** [beatmasterRS/skin.arctic.zephyr.mod#912]
<img width="1728" alt="Bildschirm­foto 2023-01-31 um 12 27 10" src="https://user-images.githubusercontent.com/24738564/215794765-1a23722d-e137-4601-82f0-678ddfb45aeb.png">

**Comment by beatmasterRS** [comment]
@kalle2000

**Comment by gwx12** [comment]
When the channel plays Artwork works for the selected channel.

<img width="1728" alt="Bildschirm­foto 2023-01-31 um 16 12 55" src="https://user-images.githubusercontent.com/24738564/215800074-a69e94e2-2abd-43e0-ab9c-f93c40cd2bcd.png">

<img width="1728" alt="Bildschirm­foto 2023-01-31 um 16 16 05" src="https://user-images.githubusercontent.com/24738564/215800129-e1211a35-7c9d-4c06-a1db-8c776c033e55.png">

**Comment by kalle2000** [comment]
@gwx12 Did you try 2.0.1 already? Is the issue occurring with that version still?

**Comment by gwx12** [comment]
Yes with 2.0.1 still not show on Homescreen, have tried to clear the artwork.

<img width="1728" alt="Bildschirm­foto 2023-02-01 um 20 42 15" src="https://user-images.githubusercontent.com/24738564/216146949-d69572db-c99d-4e23-86dc-03deb0c7d9cc.png">
<img width="1728" alt="Bildschirm­foto 2023-02-01 um 20 40 43" src="https://user-images.githubusercontent.com/24738564/216146977-5f71ca45-95a2-4c2d-8758-9d0858dcb4ba.png">

<img width="1728" alt="Bildschirm­foto 2023-02-01 um 20 42 32" src="https://user-images.githubusercontent.com/24738564/216147019-e6e7d335-8ae7-495e-9cee-7a42a55cd43d.png">

**Comment by kalle2000** [comment]
@gwx12 please try this fix and let me know if its working for you

**Comment by gwx12** [comment]
@kalle2000  no change, is there something in the settings that i am missing ?

**Comment by kalle2000** [comment]
Please provide screenshots how you configured your "LIVE-TV" menu.

**Comment by gwx12** [comment]
<img width="1728" alt="Bildschirm­foto 2023-02-04 um 17 07 01" src="https://user-images.githubusercontent.com/24738564/216777809-fbdcd912-35ab-4ed2-abc4-78994ee657c9.png">

<img width="1728" alt="Bildschirm­foto 2023-02-04 um 17 07 26" src="https://user-images.githubusercontent.com/24738564/216777929-cf2da3f9-5bae-4d56-a796-0909f8f78cd2.png">

<img width="1728" alt="Bildschirm­foto 2023-02-04 um 17 09 36" src="https://user-images.githubusercontent.com/24738564/216777826-74f53ff4-bb39-4d79-b108-2127a4c166e3.png">

<img width="1728" alt="Bildschirm­foto 2023-02-04 um 17 10 47" src="https://user-images.githubusercontent.com/24738564/216777834-ccfcb86e-41b6-4663-94a8-b88dc3e6312b.png">

<img width="1728" alt="Bildschirm­foto 2023-02-04 um 17 10 52" src="https://user-images.githubusercontent.com/24738564/216777841-32222602-1b57-4a17-b6a6-cb53567ae188.png">

**Comment by kalle2000** [comment]
@gwx12 can you please try to select "clean & minimal" as "Home menu style" in the skin settings? Any changed behavior?

**Comment by gwx12** [comment]
@kalle2000 nothing changed.

**Comment by kalle2000** [comment]
It looks like PVR Artwork is not supporting the Home screen by default. You can change that by changing the _main.py_ file in _/storage/.kodi/addons/script.module.pvr.artwork_ :+1: 

`content_types = dict({'MyPVRChannels.xml': 'ListItem', 'MyPVRGuide.xml': 'ListItem', 'DialogPVRInfo.xml': 'ListItem',
                      'MyPVRRecordings.xml': 'Container(50).ListItem', 'MyPVRTimers.xml': 'Container(50).ListItem',
                      'MyPVRSearch.xml': 'Container(50).ListItem', 'DialogPVRChannelsOSD.xml': 'Container(11).ListItem',
                      'DialogPVRChannelGuide.xml': 'Container(11).ListItem'})`

into:

`content_types = dict({'MyPVRChannels.xml': 'ListItem', 'MyPVRGuide.xml': 'ListItem', 'DialogPVRInfo.xml': 'ListItem',
                      'MyPVRRecordings.xml': 'Container(50).ListItem', 'MyPVRTimers.xml': 'Container(50).ListItem',
                      'MyPVRSearch.xml': 'Container(50).ListItem', 'DialogPVRChannelsOSD.xml': 'Container(11).ListItem',
                      'DialogPVRChannelGuide.xml': 'Container(11).ListItem', 'Home.xml': 'Container(30111).ListItem'})`

30111 work if your PVR is the first entry in the menu. In case its second, its 30121.

**Comment by gwx12** [comment]
Thank you for your effort, Artwork works now. 👍

Is it possible to display the channel Logo if there is no Artwork?
